cmd/limactl/shell.go

```diff
@@ -49,6 +51,7 @@ func newShellCommand() *cobra.Command {
 
 	shellCmd.Flags().String("shell", "", "shell interpreter, e.g. /bin/bash")
 	shellCmd.Flags().String("workdir", "", "working directory")
+	shellCmd.Flags().BoolP("exit-session", "e", false, "exit a ssh session for the instance")
```

Comment:
I'd prefer to reserve `e` for now.  (Potentially for something similar to `ssh -e`)
Can be revisited later.